debian/control

```diff
@@ -0,0 +1,37 @@
+Source: ultrastardx
+Section: games
+Priority: optional
+Maintainer: Markus Meier <memcopy@epic-bytes.net>
+Build-Depends: debhelper (>= 10),
+ autotools-dev,
+ fpc (>= 3.0.0),
+ libpng-dev,
+ libfreetype6-dev,
+ libsqlite3-dev,
+ libsdl2-dev,
+ libsdl2-image-dev,
+ liblua5.3-dev,
+ libavcodec-dev,
+ libavformat-dev,
+ libavutil-dev,
+ portaudio19-dev,
+ libswscale-dev,
+ libprojectm-dev,
+ libpcre3-dev
+Standards-Version: 3.9.8
+Homepage: https://usdx.eu
+
+Package: ultrastardx
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends},
+         fonts-dejavu-core, fonts-freefont-ttf
+Recommends: projectm-data,
+            ttf-bitstream-vera
+Suggests: fonts-wqy-microhei
```

Comment:
Suggests: libopencv-highgui-dev, libopencv-imgproc-dev, libopencv-core-dev

The symlinks from the -dev packages are dlopen'ed at runtime for webcam support.
Though for me it currently locks up after displaying the first frame.

Reply:
Ok I didn't know that. I also don't have a webcam so I didn't notice that problem. It's not nice that someone has to install the development packages to get this stuff working.

Reply:
You could try to add a patch for Debian that uses objdump or readelf during configure to extract the SONAME of the OpenCV libraries.